src/toil/provisioners/abstractProvisioner.py

```diff
@@ -497,30 +501,37 @@ def addSSHRSAKey(self, keyData: str):
 
             self.sshPublicKeys.append("ssh-rsa " + keyData)
 
-
-        def toCloudConfig(self) -> str:
+        def toIgnitionConfig(self) -> str:
             """
-            Return a CloudConfig configuration describing the desired config.
+            Return an Ignition configuration describing the desired config.
             """
 
             # Define the base config
             config = {
-                'write_files': self.files,
-                'coreos': {
-                    'update': {
-                        'reboot-strategy': 'off'
-                    },
+                'ignition': {
+                    'version': '2.2.0'
```

Comment:
If we want to use compression we might need to bump this version. Also where can we find the version 2.2 docs? Maybe the code should have a link to them?